src/app/page/Recipient/Details.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { ReactNode, useState, useContext } from "react";
+import { Typography, Button, Box, Divider, TextField } from "@material-ui/core";
+import styled from "styled-components";
+
+import Snackbar from "../../component/Snackbars";
+import { MissionInterface, MissionStatus as Status } from "../../model/schema";
+import { Mission, useOrganization } from "../../model";
+
+export default function Details({ mission }: { mission: MissionInterface }) {
+  const org = useOrganization();
+  return (
+    <>
+      <Divider />
+      <Box margin="1rem">
+        {mission.deliveryType === "delivery" ? (
+          <>
+            <DetailSection
+              header="Address"
+              content={
+                <Typography variant="body1" gutterBottom>
+                  {mission.deliveryLocation.address}
```

Comment:
mission?.

Reply:
I can, but typescript guarantees that mission will exist here

Reply:
We probably have bigger issues, if a required property is missing 😉 should trust TypeScript to enforce the requirement.